pages/settings/index.tsx

```diff
@@ -348,14 +348,14 @@ const SettingsPage = () => {
           </div>
         )}
 
-        <div className="flex h-fit flex-row justify-between bg-light-bg px-3 py-[15px] dark:bg-dark-bg">
+        {/* <div className="flex h-fit flex-row justify-between bg-light-bg px-3 py-[15px] dark:bg-dark-bg">
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, atm restricting mint changes as it could cause tokens to be sent with no information on the mint. I'll add that functionality back in a separate pr.